rel/overlay/etc/vm.args

```diff
@@ -60,3 +60,24 @@
 
 # Set maximum SSL session lifetime to reap terminated replication readers
 -ssl session_lifetime 300
+
+# TLS distribution
+#-pa <%= @tls_dir %>
+#-proto_dist cloudant
```

Comment:
Nothing in the Apache repo should reference cloudant this explicitly.....

Reply:
Removed